brainstormforce/ultimate-addons-for-gutenberg
# Fix: Table of content -  Show Scroll To Top height and background color not applying issue in the editor.

### Description

- Card - https://ua-gutenberg.atlassian.net/jira/software/projects/UAG/boards/2/backlog?selectedIssue=UAG-344

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop TOC add styling to block.
- Enable scroll to top and styling to it.
- Check all is working fine or not.
- Check all possible cases you have on your list.

### Checklist:
